src/components/App/Pages/Portfolio/PortfolioBody/PortfolioDescription/index.js

```diff
@@ -30,18 +32,26 @@ const PortfolioDescription = () => {
     }
     return null
   }
+
+  // Look for spaces between parentheses and replace them with %20
+  const fixSpacesRegExp = /\s+(?=[^()]*\))/gm
+  // Fix issue with inserting "\" when file ends in newline.
+  const fixEndLinesRegExp = /\\\n/gm
+  const cleanDescription = description64 ? description64.replace(fixSpacesRegExp, '%20').replace(fixEndLinesRegExp, '\n') : ''
```

Comment:
Is this now unnecessary, now that the data has been cleaned up to eliminate %20 in the data?

Reply:
No it is still needed, the markdown editor still incorrectly converts %20 to spaces under certain cirumstances.